seqr/management/commands/lift_project_to_hg38_tests.py

```diff
@@ -0,0 +1,357 @@
+import mock
+import __builtin__
+
+from django.core.management import call_command
+from django.test import TestCase
+
+PROJECT_NAME = u'1kg project n\u00e5me with uni\u00e7\u00f8de'
+PROJECT_GUID = 'R0001_1kg'
+ELASTICSEARCH_INDEX = 'test_index'
+INDEX_METADATA = {
+                    "gencodeVersion": "25",
+                    "hail_version": "0.2.24",
+                    "genomeVersion": "38",
+                    "sampleType": "WES",
+                    "sourceFilePath": "test_index_alias_1_path.vcf.gz",
+                }
+SAMPLE_IDS = ["NA19679", "NA19675_1", "NA19678", "HG00731", "HG00732", "HG00732", "HG00733"]
+
+TRANSCRIPT_1 = {
+  'aminoAcids': 'LL/L',
+  'biotype': 'protein_coding',
+  'lof': None,
+  'lofFlags': None,
+  'majorConsequenceRank': 10,
+  'codons': 'ctTCTc/ctc',
+  'geneSymbol': 'MFSD9',
+  'domains': [
+    'Transmembrane_helices:TMhelix',
+    'PROSITE_profiles:PS50850',
+  ],
+  'canonical': 1,
+  'transcriptRank': 0,
+  'cdnaEnd': 421,
+  'lofFilter': None,
+  'hgvs': 'ENSP00000258436.5:p.Leu126del',
+  'hgvsc': 'ENST00000258436.5:c.375_377delTCT',
+  'cdnaStart': 419,
+  'transcriptId': 'ENST00000258436',
+  'proteinId': 'ENSP00000258436',
+  'category': 'missense',
+  'geneId': 'ENSG00000135953',
+  'hgvsp': 'ENSP00000258436.5:p.Leu126del',
+  'majorConsequence': 'inframe_deletion',
+  'consequenceTerms': [
+    'inframe_deletion'
+  ]
+}
+TRANSCRIPT_2 = {
+  'aminoAcids': 'P/X',
+  'biotype': 'protein_coding',
+  'lof': None,
+  'lofFlags': None,
+  'majorConsequenceRank': 4,
+  'codons': 'Ccc/cc',
+  'geneSymbol': 'OR2M3',
+  'domains': [
+    'Transmembrane_helices:TMhelix',
+    'Prints_domain:PR00237',
+  ],
+  'canonical': 1,
+  'transcriptRank': 0,
+  'cdnaEnd': 897,
+  'lofFilter': None,
+  'hgvs': 'ENSP00000389625.1:p.Leu288SerfsTer10',
+  'hgvsc': 'ENST00000456743.1:c.862delC',
+  'cdnaStart': 897,
+  'transcriptId': 'ENST00000456743',
+  'proteinId': 'ENSP00000389625',
+  'category': 'lof',
+  'geneId': 'ENSG00000228198',
+  'hgvsp': 'ENSP00000389625.1:p.Leu288SerfsTer10',
+  'majorConsequence': 'frameshift_variant',
+  'consequenceTerms': [
+    'frameshift_variant'
+  ]
+}
+TRANSCRIPT_3 = {
+  'aminoAcids': 'LL/L',
+  'biotype': 'nonsense_mediated_decay',
+  'lof': None,
+  'lofFlags': None,
+  'majorConsequenceRank': 10,
+  'codons': 'ctTCTc/ctc',
+  'geneSymbol': 'MFSD9',
+  'domains': [
+    'Transmembrane_helices:TMhelix',
+    'Gene3D:1',
+  ],
+  'canonical': None,
+  'transcriptRank': 1,
+  'cdnaEnd': 143,
+  'lofFilter': None,
+  'hgvs': 'ENSP00000413641.1:p.Leu48del',
+  'hgvsc': 'ENST00000428085.1:c.141_143delTCT',
+  'cdnaStart': 141,
+  'transcriptId': 'ENST00000428085',
+  'proteinId': 'ENSP00000413641',
+  'category': 'missense',
+  'geneId': 'ENSG00000135953',
+  'hgvsp': 'ENSP00000413641.1:p.Leu48del',
+  'majorConsequence': 'frameshift_variant',
+  'consequenceTerms': [
+    'frameshift_variant',
+    'inframe_deletion',
+    'NMD_transcript_variant'
+  ]
+}
+
+ES_VARIANTS = [
```

Comment:
Rather than copy/ pasting all these constants, there is a file called test_utils.py where a bunch of shared data is stored (things like expected family fields). Could you move these there instead?

Reply:
Can I re-use the constants defined in seqr/utils/elasticsearch/es_utils_tests.py? If yes, then I don't have to do any copying.

Reply:
If this was a direct copy-paste and they are the same there shouldn't be an issue of moving the data to test)utils and then importing the same dictionary in the different tests

Reply:
The data have been moved into the file seqr/views/utils/test_utils.py. Is the right file to put the data?

Reply:
yes, but then you should remove it from the file it was originally in

Reply:
Removed from the original file.

Reply:
The original file seqr/utils/elasticsearch/es_utils_tests.py was updated.